lumigator/backend/backend/config_templates.py

```diff
@@ -99,7 +99,7 @@
         "use_fast": "{use_fast}",
         "trust_remote_code": "{trust_remote_code}",
         "torch_dtype": "{torch_dtype}",
-        "max_length": 500
+        "max_new_tokens": 500
```

Comment:
Since we have this option in `HfPipelineConfig`, I would change it to:

```suggestion
        "max_new_tokens": "{max_new_tokens}"
```

Reply:
Done, removed from here and keep the default of 500 in HfPipelineConfig. This allows user to pass a different value if need be.